scripts/MakeApiBaselines.ps1

```diff
@@ -11,11 +11,11 @@ if ($PSVersionTable.PSVersion.Major -lt 6) {
 }
 
 $Project = $PSScriptRoot + "/../eng/Tools/ApiChief/ApiChief.csproj"
-$Command = $PSScriptRoot + "/../artifacts/bin/ApiChief/Debug/net8.0/ApiChief.exe"
+$Command = $PSScriptRoot + "/../artifacts/bin/ApiChief/Debug/net8.0/ApiChief.dll"
 
 Write-Output "Building ApiChief tool"
 
-& dotnet build $Project --nologo --verbosity q
+& .dotnet\dotnet build $Project --nologo --verbosity q
```

Comment:
Shouldn't it use `$PSScriptRoot` instead of the current folder?

Reply:
The script the same way on the line 22 `-Path src/Libraries` assumes that we are executing it from the root folder of the repository, therefore I though it was safe to assume the same.

Reply:
I can update the script to use `$PSScriptRoot` everywhere.

Reply:
There's a mix right now, see line 25 too, so maybe good to use it everywhere, it's better than removing it. I know it's unrelated to your PR though.

Reply:
Updated.